recipes/kustomization-oss-all.yml

```diff
@@ -0,0 +1,40 @@
+#-------------------------------------------------------------------------------
+# Minimum Starting Point Recipe with Armory Scale Agent.
+#
+# Self contained Spinnaker installation with no external dependencies and no
+# additional configuration needed. This is intended as a starting point for any
+# kubernetes cluster. Not for production use.
+#
+# Features:
+# - Spinnaker Operator
+# - One Kubernetes account (Spinnaker's own cluster) using the Scale Agent
+# - Spinnaker authentication disabled
+# - Self hosted minio as a persistent storage
+# - Self hosted redis backend for caching and temporal storage of services
+#-------------------------------------------------------------------------------
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: spinnaker
+
+resources:
+  - infrastructure/mysql.yml           # (Optional, not for production use). MySQL server needed if using SQL backend in Spinnaker
+
+components:
+  - operator/oss/cluster                  # Installs the OSS Operator
+  - core/base                             # Defines base configuration across the cluster
+  - core/persistence/in-cluster           # Enables persistence using MinIO in-cluster
+  - targets/kubernetes/scale-agent        # Enables the plugin version of the Scale Agent
+
+patchesStrategicMerge:
+  - core/patches/oss-version.yml          # Sets the OSS version to use
+  - persistence/patch-sql-clouddriver.yml # setup clouddriver to use SQL for storage
```

Comment:
Not super urgent but it would probably make sense to merge this into the in-cluster persistence patch. That way minio/mysql are considered the "in-cluster" experience and if they need something more production grade they remove it for s3/aurora in AWS (for example)